packages/workspace/src/browser/browser-workspace-frontend-module.ts

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
why the name change ? isn't the directory enough ?

Reply:
See `browser-menu-module.ts` and `electron-menu-module.ts`.

Reply:
seems like it's `browser-menu-module.ts`  and `electron-menu-module.ts` that are wrong ? everything else is browser/something-frontend-module ?

Reply:
Did not find another place where we have customized bindings for electron and browser. Please show another example.

Reply:
monaco is an example. `monaco-frontend-module` `monaco-electron-module` 

But see: https://github.com/theia-ide/theia/wiki/Code-Organization

`Within the top level folders, that indicate what the contained code is about, we separate between the following platforms:`

Why separate in the filename what we already did in the directory?